pkg/operator/provisioner/provisioner_test.go

```diff
@@ -40,6 +40,8 @@ func TestProvisionImage(t *testing.T) {
 	clientset := test.New(3)
 	namespace := "ns"
 	configDir, _ := ioutil.TempDir("", "")
+	os.Setenv("POD_NAMESPACE", "rook-system")
+	defer os.Setenv("POD_NAMESPACE", "")
```

Comment:
same here, i wonder if restoring the original value is smart

Reply:
same as above. I'm not seeing why a variable would be set between unit tests